alacritty/src/display.rs

```diff
@@ -198,6 +202,11 @@ impl Display {
             api.clear(background_color);
         });
 
+        // Enable subpixel anti-aliasing.
+        #[cfg(target_os = "macos")]
+        let use_thin_strokes = config.font.use_thin_strokes();
+        Self::enable_font_smoothing(use_thin_strokes);
```

Comment:
The idea behind calling this function here is to run the code initially when Alacritty starts.